docs/readmes/faq/magma_faq.md

```diff
@@ -91,15 +91,15 @@ This section lists some of the commonly asked questions related to Magma operati
   - Login to Gateway, then run `sudo checkin_cli.py`.
 
 ### Where can I find API endpoint?
-  - If you have followed the [install guide](https://magma.github.io/magma/docs/orc8r/deploy_install), it'll be at `https://api.youdomain.com/apidocs/v1/` (the trailing slash is important).
+  - If you have followed the [install guide](https://magma.github.io/magma/docs/orc8r/deploy_install), it'll be at `https://api.youdomain.com/swagger/v1/ui`.
```

Comment:
Here and elsewhere -- let's keep the trailing slash for the monolithic endpoints